src/app/app_menus.h

```diff
@@ -69,7 +69,7 @@ namespace app {
     Widget* createInvalidVersionMenuitem();
     void applyShortcutToMenuitemsWithCommand(Menu* menu, Command* command, const Params& params, Key* key);
     void clearIdentifiedWidgets();
-
+    base::UniquePtr<Menu> m_rootMenu;
```

Comment:
Is this needed?

Reply:
I don't think so, I guess he didn't pull to the latest commit (same for the empty line)

Reply:
You are right, it isn't